<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1361</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(254,254,254)</string>
  </property>
  <widget class="QLabel" name="long_description">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>551</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font-weight: bold;
font: 12pt &quot;HelveticaNeueCyr&quot;;</string>
   </property>
   <property name="text">
    <string>Длинное описание</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <property name="indent">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
color: black;
font: 28pt &quot;Futurespore Cyrillic&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <property name="indent">
    <number>1</number>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>-10</y>
     <width>151</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QLabel" name="long_description_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>551</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
font-weight: bold;
font: 12pt &quot;HelveticaNeueCyr&quot;;</string>
   </property>
   <property name="text">
    <string>Длинное описание</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <property name="indent">
    <number>3</number>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_item">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>991</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed {
border: 0px solid #000;
border-radius: 13px;
border-style: outset;
background: black;
color: rgb(254,254,254);
font-weight: bold;
font: 20pt &quot;Futurespore Cyrillic&quot;;
}
QPushButton {
border: 1px solid #000;
border-radius: 13px;
border-style: outset;
color: black;
font-weight: bold;
font: 20pt &quot;Futurespore Cyrillic&quot;;
}</string>
   </property>
   <property name="text">
    <string>Удалить из корзины</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futurespore Cyrillic</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="toolTipDuration">
    <number>1</number>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRepeatInterval">
    <number>1</number>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="price">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:pressed {
border: 0px solid #000;
border-radius: 13px;
border-style: outset;
background: black;
color: rgb(254,254,254);
font-weight: bold;
font: 20pt &quot;Futurespore Cyrillic&quot;;
}
QPushButton {
border: 1px solid #000;
border-radius: 13px;
border-style: outset;
color: black;
font-weight: bold;
font: 20pt &quot;Futurespore Cyrillic&quot;;
}</string>
   </property>
   <property name="text">
    <string>цена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
